backend/main/sheets_api/SheetsAPIHandler.java

```diff
@@ -134,14 +134,14 @@ public void createMonthRows(String monthYear) {
             values.add(Arrays.asList(date));
         }
         int lastRow = dates.size() + 1;
-        String range = "A2:A" + lastRow;
+        String range = "'" +  monthYear + "'!A2:A" + lastRow;
         updateSpreadsheetValues(range, values);
     }
 
     public void createExpensesColumns(List<String> expenses) {
         int lastColumn = expenses.size() + 1;
         char lastColChar = ColumnUtils.getColumnForNumber(lastColumn);
-        String range = "B1:" + lastColChar + "1";
+        String range = "'October 2019'!B1:" + lastColChar + "1";
```

Comment:
@sor2 This is what I need to avoid, is what you're making going to be able to be able to pass this in from where this createExpensesColumns method is called?